### ISSUE TITLE
OVC_SERV FY17 APR calculation

### ISSUE THREAD

**Issue by achafetz** [ICPI/ICPIutilities#36]
@shapaklyak pointed out that that the cumulative values for OVC_SERV total numerator are a sum of the disaggs due to MER adjustments, unlike FY18 which is just Total Numerator

> for FY18 APR, OVC SERV total numerator will equal FY18 Q4, but for APR 17 it was = (Q4 for Program Status: Active) + (Q2+Q4 for Program Status: Graduated) + (Q2+Q4 for Program Status: Transferred) + (Q2+Q4 for Program Status: Exited)

Need to think about how to make this adjustment in `add_cumulative()`

https://github.com/ICPI/ICPIutilities/blob/958f130fffeb58c955669f2ed1de569e87fe9924/R/add_cumulative.R#L76-L78

**Comment by achafetz** [comment]
The code would look like this, appended onto `add_cumulative()` around line 38

```
if("fy2017apr" %in% names(df_mer)){
  
  #filter for just OVC ProgramStatus to convert and then aggregate
  df_ovc <- df_mer %>% 
    dplyr::filter(indicator == "OVC_SERV", standardizeddisaggregate == "ProgramStatus")
  
  #change program status to match Total Numerator
  df_ovc <- df_ovc %>% 
    dplyr::mutate(disaggregate = "Total Numerator",
                  standardizeddisaggregate = "Total Numerator",
                  categoryoptioncomboname = "default",
                  agefine = NA,
                  otherdisaggregate = NA)
  
  #grouping variables for aggregation
  lst_meta <- df_ovc %>% 
    select_if(is.character) %>% 
    names()
  
  #aggregate fy2017apr  
  df_ovc <- df_ovc %>%
    dplyr::group_by_at(vars(lst_meta)) %>% 
    dplyr::summarise_at(vars(fy2017apr), ~ sum(., na.rm = TRUE)) %>% 
    dplyr::ungroup()
  
  #remove fy2017apr and merge on actual OVC numerator
  df_mer <- df_mer %>%
    dplyr::mutate(fy2017apr = ifelse((indicator == "OVC_SERV" & standardizeddisaggregate == "Total Numerator"), NA, fy2017apr)) %>% 
    dplyr::bind_rows(df_ovc)
  rm(df_ovc)
}
```

**Comment by achafetz** [comment]
@shapaklyak this ended up being a finicky fix, but its done now and just clearing the CI pull clearance. It will be fixed in [version 1.0.22](https://github.com/ICPI/ICPIutilities/blob/master/NEWS.md) when the pull completes.
